ua/resolve/resolve.go

```diff
@@ -0,0 +1,67 @@
+package resolve
+
+import (
+	"sync"
+
+	"github.com/gopcua/opcua"
+	"github.com/gopcua/opcua/errors"
+)
+
+// Lookup returns the id from a list of namespace URLs or an error.
+// The values are not parsed and are compared case-sensitive.
+func Lookup(s string, ns []string) (uint16, error) {
+	for id, nsu := range ns {
+		if nsu == s {
+			return uint16(id), nil
+		}
+	}
+	return 0, errors.Errorf("unknown namespace url %s", s)
+}
```

Comment:
This will be called as `resolve.Lookup` where we are trying to get numeric id of a namespace.

Shouldn't the string "namespace" into the invocation?

Something like "namespace.Lookup()"?

Reply:
good point. Maybe we can call it `nsu`